scripts/spawn_notebook.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+USERNAME=${JUPYTERHUB_USER}
+
+echo "Starting Jupyter Notebook for user: $USERNAME"
+cd $JUPYTERHUB_USER_HOME/$USERNAME
+
+# Start the notebook server with current user
+exec jupyterhub-singleuser "$@"
```

Comment:
This looks like a shell injection vulnerability

Reply:
👍

Reply:
:+1: